### PR TITLE
Document our git workflow

### PR DESCRIPTION
**Pull request by palourde**
Signed-off-by: Simon Plourde <simon@sensu.io>

## What is this change?

This PR documents our git workflow in the CONTRIBUTING document.

Here's the image referred:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -178,6 +178,18 @@ Examples:
 
 - Refactored how Checks are stored in Etcd, `sensu-backend migrate` is required to upgrade
 
+## Git Workflow
+
+Our git workflow is largely inspired by [GitHub Flow](https://guides.github.com/introduction/flow/) and [Oneflow](https://www.endoflineblog.com/oneflow-a-git-branching-model-and-workflow) but adapted to our reality and our needs.
+
+Here are the hightlights:
```
Comment (jamesdphillips) [inline CONTRIBUTING.md]: highlights

**Comment by portertech** [comment]
best-thing-ever-hand

**Comment by portertech** [comment]
wind_beneath

**Pull request merged**